libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/custom-stepper.component.html

```diff
@@ -0,0 +1,116 @@
+<div class="grid-row">
+  <div class="desktop:grid-col-4 tablet-lg:grid-col-12 mobile-lg:grid-col-12 margin-bottom-3 margin-right-2">
+    <sds-side-toolbar [responsiveButtonText]="'Select Step'" (responsiveDialog)="onDialogOpen($event)">
+      <ng-template #toolbarContent>
+        <div class="sds-card" *ngIf="!!responseDialog">
```

Comment:
Double negative? Does just `ngIf="responseDialog"` not work?

Reply:
I just copied this part of code from the older documentation, but I removed the double negative because it wasnt needed anyways.

Reply:
Still here as of your latest push.

Reply:
@shayan-roshan are you still planning on removing this?

Reply:
removed